[BLOG] Feb community update (#173)
pulsar-edit/pulsar-edit.github.io: docs/blog/20230201-Daeraxa-FebUpdate.md

---
title: February Update
author: Daeraxa
date: 2023-02-01
category:
  - news
  - log
tag:
  - update
---

What has the Pulsar team and community been up to lately?

<!-- more -->

# Community Update

Hi everyone, welcome to our first post in what we hope will be a regular (or at
least semi-regular) set of updates from the Pulsar team to let you know what
is going on in the background. We see updates, work and improvements to the
entire Pulsar ecosystem nearly every single day but many of these either aren't
very obvious or requires a very close following of the various communication
channels.  
This update format is designed to let people know of our major wins and exciting
updates of things we are working on so you can share in the progress we are
making.  
We hope you enjoy reading this and getting to know what is going on with the
project and as ever we will see you among the stars!

## Tree-sitter

Those who have been keeping up with the development of Pulsar know that one of

> our major goals is to move upgrade to newer versions of Electron. Unfortunately

Reviewer: ```suggestion
our major goals is to upgrade to newer versions of Electron. Unfortunately
```

Author: Whoops, stream of consciousness made it in. I'm awful at reviewing my own stuff

Reviewer: Hey still less issues then when I write something out, so a win by my standards aha

this comes with quite a few challenges, one of which is the implementation of

> [Tree-sitter](https://tree-sitter.github.io/tree-sitter/), a language parsing

Reviewer 2: Not sure why, but "language parsing library" sounds a bit... weird to me? Not sure what is a better way to say it though, so I would be fine if its left as is

Author: I could take from the tree-sitter description instead:
> Tree-sitter is a parser generator tool and an incremental parsing library.
> An incremental parsing system for programming tools 

I can change to "incremental parsing library" but i'm not sure people would understand what that meant.

Reviewer 2: hmmmmm, "A library used for syntax highlighting"?

Author: yeah that sounds better

contact the owner of each package and ask for permission to host them.
The good news is that this has been well received and we have managed to
preserve more than a few of them.

## Donations and binary signing

We have had an awful lot of generous donations sent to us recently. In
particular we have to give a massive shout out to **_@SubAtomic_** who has
donated more money to the project alone than we thought we get in a whole year.

**_@anonCoffee_** also gave us a very generous donation and was also the first
sponsor on our new [GitHub Sponsorship](https://github.com/sponsors/pulsar-edit)
which is an alternative should you want to donate by a platform other than our
existing [Open Collective](https://opencollective.com/pulsar-edit).

We also now have a webhook set up for the GitHub sponsors so now when we get
a donation we can properly thank them as it pops up on our Discord `#donations`
channel.

A huge thank you to literally all of our other donators on both platforms, we
appreciate everything you give and make so much of what we want to do, possible.

Our first big expenditure we voted for (outside of ongoing hosting costs) will

> be to pay for licences from Apple and Microsoft so that we can finally sign our

Reviewer: ```suggestion
be to pay for Code Signing Certificates from Apple and Microsoft so that we can finally sign our
```

Just updating this to what feels a bit more accurate.

Author: Makes sense, exactly what I was after

binaries to prevent the issue of new users being told that they are broken,
dangerous or may contain harmful code. [@Meadowsys] has already started on the
macOS process of this so with any luck the need to run a special command just
to run our software will be a thing of the past.

## Matrix space

[@kaosine] has been working to set us up a [Matrix](https://matrix.org/) space for

> an alternative who would prefer to use an open source alternative to our main

Reviewer: ```suggestion
an alternative for people who would prefer to use an open source alternative to our main
```

Author: Ugh, I actually corrected that once but then forgot to sync that change between my computers, good catch.
